### DISCUSSION TITLE
osu!lazer f2 crash

### DISCUSSION THREAD

**Discussion by Superbreadex** [ppy/osu#17343] [Q&A] [answered]
Hello, 
Recently installed lazer and had a ton of error messages and crashed the game
I managed to do it many times 
I imported all maps from stable and it went ok i guess
But then went into map menu and pressed f2 3 to 4 times and boom 
I have around all ranked and loved maps in the game (~90k)
Here are the logs 
[runtime.log](https://github.com/ppy/osu/files/8309207/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8309208/updater.log)
[database.log](https://github.com/ppy/osu/files/8309209/database.log)
[network.log](https://github.com/ppy/osu/files/8309210/network.log)
[performance.log](https://github.com/ppy/osu/files/8309211/performance.log)

![image](https://user-images.githubusercontent.com/91629408/159121492-70619af7-0ee3-4643-bfb7-7e92c2d4699e.png)

The error probably comes from the fact that my hard drive is full, also i would like to know if it is possible to ask which hard drive to use at the installation of the game, or tell me that i don't have enough space on the disk

**Comment by bdach** [comment, marked as answer]
you are correct. the issue is lack of disk space.

> 2022-03-19 12:29:05 [error]: System.IO.IOException: Espace insuffisant sur le disque. : 'C:\Users\bapti\AppData\Roaming\osu\cache\fonts\e8b2344cb1fe6bd3aa8837f99703672f#c34954f9025281d1a4a9f081cea925a5#1024#1024'

you can change the folder in which beatmap files are stored via game settings (see "change folder location" option).

**Comment by peppy** [comment]
Will add that this should be pre-empted via https://github.com/ppy/osu/issues/5997.
